app/views/audio_recordings/downloader/download_audio_files.ps1.erb

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env pwsh
+
+# Requires -Version 7
+<#
+
+.DESCRIPTION
+
+  Downloads a set of recordings from an acoustic workbench.
+  Will work on Windows, Linux, or MacOSX with PowerShell Core.
+  See https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.1
+  for installation instructions.
+
+  Version <%= @model.app_version %> from <%= @model.workbench_url %>.
+
+.EXAMPLE
+
+PS> ./download_audio_files.ps1
+
+.EXAMPLE
+
+$ chmod +x download_audio_files.ps1
+$ ./download_audio_files.ps1
+
+#>
+
+param(
+  # Where to download the recordings. Defaults to the present working directory if not specified.
+  $target = $null,
+  # The username to use to login to the workbench. This value's default value is templated when the script is generated.
+  $user_name = $null,
+  # The auth token to use to login to the workbench.
```

Comment:
I consider that advanced usage.  I added `You only need to supply an auth token if you want to log in without a password.`. People can ask about it otherwise. https://github.com/QutEcoacoustics/workbench-client/issues/1653

BTW the docs for this script will be on the workbench-client website.